bitnami/joomla/Chart.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: mariadb
+  repository: https://charts.bitnami.com/bitnami
+  version: 8.0.7
+- name: common
+  repository: https://charts.bitnami.com/bitnami
+  version: 0.10.0
+digest: sha256:dffbd0242679c5af046f5701d9078fd8efcbfe8f81d2cc4054c200bb0a03b696
+generated: "2020-11-11T20:30:50.056218268Z"
```

Comment:
This file is going to be automatically updated by the pipeline once executed the Docker Image job using this PR as parameter